ducktape/decorate.py

```diff
@@ -0,0 +1,229 @@
+# Copyright 2015 Confluent Inc.
```

Comment:
The module name sounds like it'll just end up being a dumping ground for any sort of decorator. Rename to parametrize?